### ISSUE TITLE
Configure-Opsman Task erroneously prints help

### ISSUE THREAD

**Issue by daviddob** [pivotal/docs-platform-automation#37]
**Describe the bug**
When using the configure-opsman task it will print the help for `vm-lifecycle` before executing the proper commands successfully.

This is due to the `om vm-lifecycle -v` call that can be found here https://github.com/pivotal/docs-platform-automation/blob/e83acf9779bd265a0bcfe175ac3777361529de8e/tasks/configure-opsman.sh#L5

**To Reproduce**
1. Use the configure-opsman task in any pipeline

**Expected behavior**
The configure-opsman task to function without printing help unnecessarily or an unrelated command

**Screenshots**
N/A

```
selected worker: 0b2d37d2-740e-486f-8ddc-959672ba1b74
5.0.15
Usage:
  om [OPTIONS] vm-lifecycle <command>

commands to manage the state of the Ops Manager VM. Requires the cli of the desired IAAS to be installed.

Application Options:
      --ca-cert=               OpsManager CA certificate path or value [$OM_CA_CERT]
  -c, --client-id=             Client ID for the Ops Manager VM (not required for unauthenticated commands) [$OM_CLIENT_ID]
  -s, --client-secret=         Client Secret for the Ops Manager VM (not required for unauthenticated commands) [$OM_CLIENT_SECRET]
  -o, --connect-timeout=       timeout in seconds to make TCP connections (default: 10) [$OM_CONNECT_TIMEOUT]
  -d, --decryption-passphrase= Passphrase to decrypt the installation if the Ops Manager VM has been rebooted (optional for most commands) [$OM_DECRYPTION_PASSPHRASE]
  -e, --env=                   env file with login credentials
  -p, --password=              admin password for the Ops Manager VM (not required for unauthenticated commands) [$OM_PASSWORD]
  -r, --request-timeout=       timeout in seconds for HTTP requests to Ops Manager (default: 1800) [$OM_REQUEST_TIMEOUT]
  -k, --skip-ssl-validation    skip ssl certificate validation during http requests [$OM_SKIP_SSL_VALIDATION]
  -t, --target=                location of the Ops Manager VM [$OM_TARGET]
      --trace                  prints HTTP requests and response payloads [$OM_TRACE]
  -u, --username=              admin username for the Ops Manager VM (not required for unauthenticated commands) [$OM_USERNAME]
      --vars-env=              load vars from environment variables by specifying a prefix (e.g.: 'MY' to load MY_var=value) (default: OM_VAR) [$OM_VARS_ENV]
  -v, --version                prints the om release version

Help Options:
  -h, --help                   Show this help message

Available commands:
  create-vm                   Create VM for Ops Manager to a given IaaS
  delete-vm                   Delete VM from a given IaaS
  export-opsman-config        Exports an opsman.yml for an existing Ops Manager VM
  prepare-tasks-with-secrets  Modifies task files to include config secrets as environment variables
  upgrade-opsman              Deletes the old opsman vm given an exported installation, bring up a new vm, and import that installation
+ vars_files_args=("")
+ for vf in ${VARS_FILES}
+ vars_files_args+=("--vars-file ${vf}")
+ for vf in ${VARS_FILES}
+ vars_files_args+=("--vars-file ${vf}")
...
(rest of the successful task here)
```

**Desktop (please complete the following information):**
N/A

**Smartphone (please complete the following information):**
N/A

**Additional context**
```
om --version
6516c1a327f7bb7ede88c857e5b4d0d58f27f5bc-2021-01-26T10:53:54-07:00
```
Concourse v6.7.1
Platform Automation v5.0.15

**Comment by daviddob** [comment]
If this is not the correct place to submit issues to the tasks themselves please let me know where I should be directing future issues/PRs. Thanks.

**Comment by jaristiz** [comment]
Hi @daviddob,

We took a look at the issue and are planning on working on it, I'll let you know when we deploy a new version of the tooling with the fix.

Regards,

**Comment by Spimtav** [comment]
Hello @daviddob,

We just released [v5.0.18](https://github.com/pivotal/docs-platform-automation/releases/tag/v5.0.18) and [v4.4.25](https://github.com/pivotal/docs-platform-automation/releases/tag/v4.4.25) which fix this issue.  Sorry for the long delay!!

Ur Lobstery Pal ©,
claire
